### ISSUE TITLE
check POS benchmark/training behavior

### ISSUE THREAD

**Issue by mssammon** [IllinoisCogComp/illinois-cogcomp-nlp#251]
Verify that there is a train/dev/test split specified and used by the training/benchmark test script(s) (class POSTrain.java)

**Comment by mssammon** [comment]
@cgraber any questions/progress on this? -- the chunker will need something similar, so you could coordinate with @qiangning who has been working on chunker-related tasks.

**Comment by cgraber** [comment]
I haven't had the time yet; I'll try to take a look at it this weekend.

**Comment by cgraber** [comment]
Train/dev/test split is specified in the file POSConfigurator.java; if POSTrain.java is run without arguments, the training/dev splits are automatically used as the training data.

The main methods of both TestPOSModels.java and TestPOS.java use the test split.

The script train.sh uses the default training set for POSTrain.java; Benchmark.sh uses the default test set for TestPOS.java.

Is there anything else I need to check for this?

**Comment by mssammon** [comment]
Thanks. If you haven't already done so, please just verify that the scripts run.

**Comment by cgraber** [comment]
Yes they do (assuming you have already compiled/packaged the POS project).
